src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +13,34 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        int distanceX = toX - robot.getX();
```

Comment:
Separate this blocks to smaller methods. It's a good practice to have some small methods instead of single and giant :)